src/main/java/org/apache/accumulo/examples/reservations/ARS.java

```diff
@@ -301,14 +301,15 @@ public void run() {
       } else if (tokens[0].equals("quit") && tokens.length == 1) {
         break;
       } else if (tokens[0].equals("connect") && tokens.length == 6 && ars == null) {
-        try (AccumuloClient client = Accumulo.newClient().to(tokens[1], tokens[2])
-            .as(tokens[3], tokens[4]).build()) {
-          if (client.tableOperations().exists(tokens[5])) {
-            ars = new ARS(client, tokens[5]);
-            reader.println("  connected");
-          } else
-            reader.println("  No Such Table");
-        }
+        // its good to close clients when possible, but it was not done here for simplicity
+        AccumuloClient client = Accumulo.newClient().to(tokens[1], tokens[2])
+            .as(tokens[3], tokens[4]).build();
+        if (client.tableOperations().exists(tokens[5])) {
+          ars = new ARS(client, tokens[5]);
+          reader.println("  connected");
+        } else
+          reader.println("  No Such Table");
```

Comment:
This is a very confusing nesting without the braces.

Reply:
The braces around the else? I didn't notice that when I was removing the try-block braces, thanks for bringing it to my attention.